packages/clerk-js/src/ui/hooks/usePasswordComplexity.ts

```diff
@@ -0,0 +1,99 @@
+import type { PasswordSettingsData } from '@clerk/types';
+import { useCallback, useMemo, useState } from 'react';
+
+import { localizationKeys, useLocalizations } from '../localization';
+
+const testComplexityCases = (password: string) => {
+  return {
+    min_length: password.length >= 8,
```

Comment:
🔧 there are some settings missing (eg `max_length`) based on the defined `PasswordSettingsData ` type.